Source/Core/Core/PowerPC/JitILCommon/IR.cpp

```diff
@@ -1254,11 +1273,25 @@ InstLoc IRBuilder::FoldBiOp(unsigned Opcode, InstLoc Op1, InstLoc Op2, unsigned
 		case FDAdd:
 		case FDSub:
 			return FoldDoubleBiOp(Opcode, Op1, Op2);
+		case Store8:
+		case Store16:
+		case Store32:
+		case StoreSingle:
+		case StoreDouble:
+		{
+			auto mem = FoldMemoryAddress(Op2);
+			return EmitTriOp(Opcode, Op1, mem.first, mem.second);
```

Comment:
Ah, That's what the intent was.

I was originally using FoldTriOp but it created a whole lot more unused Zero Constant nodes.